testharness/src/test/java/bio/terra/policy/service/policy/MutatorTest.java

```diff
@@ -1,16 +1,18 @@
 package bio.terra.policy.service.policy;
 
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsInAnyOrder;
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
 
 import bio.terra.policy.common.model.PolicyInput;
 import bio.terra.policy.testutils.LibraryTestBase;
 import com.google.common.collect.ArrayListMultimap;
 import com.google.common.collect.Multimap;
 import java.util.Set;
-import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 
-public class CombinerTest extends LibraryTestBase {
+public class MutatorTest extends LibraryTestBase {
```

Comment:
nit: name should be PolicyMutatorTest to match the java class name. Thought based on your test content, it's more of a PolicyGroupconstraintTest